frontend/src/app/page.tsx

```diff
@@ -0,0 +1,32 @@
+"use client"
+import { useSession, signIn, signOut } from "next-auth/react"
+import Button from "@components/Button"
+
+export default function Home() {
+  const { data: session } = useSession()
+
+  if (session) {
+    return (
+      <>
+        Signed in as {session?.user?.email} <br />
+        <Button accent="tertiary" onClick={() => signOut()}>
+          Sign out
+        </Button>
+      </>
+    )
+  }
+
+  return (
+    <>
+      Not signed in <br />
+      <Button
+        accent="secondary"
+        onClick={() =>
+          signIn("spotify", { callbackUrl: "http://localhost:3000" })
```

Comment:
Can this be parametrized so it goes to the home page without explicitly using the host and the port? Otherwise, this will fail when deploying in production.